apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/event/EventServiceImpl.java

```diff
@@ -75,4 +79,14 @@ public Event getById(Long id) {
         return eventRepository.findById(id).orElseThrow(EventNotFoundException::new);
     }
 
+    @Override
+    public List<EventType> getEventTypes() {
+        User user = authenticationService.getAuthenticatedUser();
```

Comment:
As it is dependent on authenticated user probably tis inforamtion should br bundled with `/auth/me` response. Name of that function (and endpoint path) suggest that it returns all allowed event types, but it returns only those that current user can create.